content/04.results.md

```diff
@@ -169,6 +170,28 @@ Our filtering criteria were intentionally stringent, so the increased estimate o
 Many of these missed annotations were for preprints posted in the 2017-2018 interval.
 As opposed to those published in 2019 and later, these preprints are old enough that they are likely to have been published but it was interesting that the rate was not observed to be higher for older preprints.
 
-### Journal Recommendations/Audience Associations
-1. Title will change once analysis is finished 
-2. Provide key figure for this section and take-home message
+### Recommending Journals Based on Preprint Representation
+
+![
+Both classifiers outperform the randomized baseline when predicting a paper's journal endpoint.
+This bargraph shows each model's accuracy in respect to predicting the training and test set.
+](https://raw.githubusercontent.com/danich1/annorxiver/6bedc581a88b8d98043d847d234d06877458d898/pmc/journal_recommendation/output/figures/knn_result.png){@fig:knn_auc}
+
+We sought to recommend journals based on the textual content of a paper.
+We trained two different classifiers (centroid classifier and paper-paper classifier) to predict the journal endpoints for already published papers. 
+We found that both clasifiers are better than our randomized baseline and that our centroid classifier acheived better accuracy on the test set compared to the paper-paper classifier (Figure {#fig:knn_auc}).
+Despite the positive performance, recommending journals remains a challenging task as both model's overall accuracy is modest at best (Figure {#fig:knn_auc}).

```

Comment:
```suggestion
There are [HOW MANY] journals in our dataset, so the baseline performance of a classifier is quite low.
We were able to achieve a substantial increase with respect to random performance at predicting the journals that a paper was published in.
However, the predictor is not perfect (Figure {#fig:knn_auc}), which we should expect because there are multiple journals that cover certain topic areas and others have a very broad set of covered topic areas.
Still, our software provides a starting point for authors to use the text of their preprints to identify potentially suitable publication venues.
```